proposed/extended-coding-style-guide.md

```diff
@@ -405,7 +440,9 @@ class ClassName
 
 Argument lists MAY be split across multiple lines, where each subsequent line
 is indented once. When doing so, the first item in the list MUST be on the
-next line, and there MUST be only one argument per line.
+next line, and there MUST be only one argument per line. A single argument being
+split across multiple lines (As might be the case with an anonymous function or
+array) does not constitute splitting the argument list itself.
 
```

Comment:
This all applies to parameters/arguments I believe. Otherwise the PSR-2 errata is wrong too.